classes.py

```diff
@@ -27,12 +27,52 @@ def draw(self):
   def erase(self):
     pygame.draw.polygon(self.game.screen, self.game.bg_colour, self.vertices)
 
+class ShootingMissiles:
+  def __init__(self, destination, missile_locations, enemy, game):
+    self.missiles = [Missile(location, game) for location in missile_locations]
+    self.destination = destination
+    self.missiles_in_place = []
+    self.enemy = enemy
+    self.is_shot = False
+    self.has_collided = False
+  
+  def update_locations(self):
+    if not self.is_shot:
+      for missile in self.missiles:
+        if missile.vertices[2].x < self.destination:
```

Comment:
This is quite ared to read since its not obvious which of the three vertices the index 2 refers to. A `Triangle` object with `top`, `left` and `right` vertices or something similar would help greatly.